CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 * [8918](https://github.com/grafana/loki/pull/8918) **salvacorts** Introduce limit to require at least a number label matchers on metric and log queries.
 * [8909](https://github.com/grafana/loki/pull/8909) **salvacorts** Requests to `/loki/api/v1/index/stats` are split in 24h intervals.
 * [8732](https://github.com/grafana/loki/pull/8732) **abaguas**: azure: respect retry config before cancelling the context
+* [8874](https://github.com/grafana/loki/pull/8874) **rfratto**: Support expoential backoff when polling unchanged files for logs.
```

Comment:
nit, but can we prepend this with `promtail`? this changelog is for the whole repo

Reply:
Are you sure? It's in the promtail header: 

![image](https://user-images.githubusercontent.com/630212/231294527-f7b35a52-6428-48fd-b726-a9041d3c62de.png)

Reply:
Yeah, good point. Should I still add the Promtail prefix?

Reply:
personally, I think so yes

Reply:
Done 👍